Classes_Homework5/modules/input.py

```diff
@@ -0,0 +1,114 @@
+import pyinputplus as pyip
+import re
+
+
+class Selection:
+    def ask_for_selection(self, label1, label2, label3):
+        """Initial input of section selection (news, ad or recipe).
+        All sections are configurable in configs.ini.
+        """
+        section = pyip.inputMenu([label1, label2, label3], numbered=True)
+        return section
+
+    def next_steps(self, msg1):
+        """Ask user whether she/he wants to proceed after section is added"""
+        decision = pyip.inputYesNo(msg1 + '\n')
+        return decision
+
+    def goodbye(self, msg2):
+        """Say goodbye to user if he/she doesn't want to proceed"""
+        print(msg2)
+
+
+class TextInput:
+    def __init__(self, error_text, max_size):
+        self.error_text = error_text
+        self.max_size = max_size
+
+    def ask_for_text(self, input_text):
+        """Ask for text input + raise error if text too long.
+        Message for input, error message and max size are configurable.
+        inputStr checks for empty string and won't allow it"""
+        proper_text = False
+        while not proper_text:
+            try:
+                user_input_text = pyip.inputStr(input_text+'\n')
```

Comment:
Looks like I can't type multiline text using it, for example:
```
Hello
World
```
 I'd probably create some custom function in order to handle that.